test/compile/mark/init.test.ts

```diff
@@ -226,6 +226,30 @@ describe('compile/mark/init', function() {
       assert.equal(model.markDef.orient, 'vertical');
     });
   });
+
+  describe('utc', function() {
```

Comment:
This describe seems out of place -- you have to describe the method name you are testing. 

In any case, seems like this has to move to normalize.